content/influxdb/cloud-dedicated/write-data/troubleshoot.md

```diff
@@ -26,32 +26,36 @@ Learn how to avoid unexpected results and recover from errors when writing to {{
 - [Troubleshoot failures](#troubleshoot-failures)
 - [Troubleshoot rejected points](#troubleshoot-rejected-points)
 
-<!-- /TOC -->
 
 ## Handle `write` responses
 
 In {{% product-name %}}, writes are synchronous.
-After InfluxDB validates the request and ingests the data, it sends a _success_ response (HTTP `204` status code) as an acknowledgement that the data is written and queryable.
-To ensure that InfluxDB handles writes in the order you request them, wait for the acknowledgement before you send the next request.
+After InfluxDB validates the request and attempts to write the data, it responds with a _success_ or _error_ message that indicates the final status of the write.
 
-If InfluxDB successfully writes all the request data to the database, it returns _success_ (HTTP `204` status code).
-The first rejected point in a batch causes InfluxDB to reject the entire batch and respond with an [HTTP error status](#review-http-status-codes).
+A success response (HTTP `204` status code) acknowledges that all data in the batch is written and queryable.
+
+If InfluxDB responds with an [HTTP error status code](#review-http-status-codes), one or more points weren't written.
+The response body contains information about [rejected points](#troubleshoot-rejected-points).
+Written points are queryable.
+
+To ensure that InfluxDB handles writes in the order you request them, wait for the response before you send the next request.
 
 ### Review HTTP status codes
 
 InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
 The `message` property of the response body may contain additional details about the error.
 Write requests return the following status codes:
 
-| HTTP response code              | Message                                                                 | Description    |
+| HTTP response code              | Response body                                                                    | Description    |
 | :-------------------------------| :---------------------------------------------------------------        | :------------- |
 | `204 "Success"`                 |                                                                         | If InfluxDB ingested the data |
-| `400 "Bad request"`             | `message` contains the first malformed line                             | If data is malformed    |
+| `400 "Bad request"`             | `line` contains the first rejected line, `message` describes rejections | If data is malformed    |
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.